mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -2980,23 +3144,27 @@ func.func @flat_transpose_index(%arg0: vector<16xindex>) -> vector<16xindex> {
 
 // -----
 
-//===----------------------------------------------------------------------===//
-// vector.load
-//===----------------------------------------------------------------------===//
-
 func.func @flat_transpose(%arg0: vector<16xf32>) -> vector<16xf32> {
   %0 = vector.flat_transpose %arg0 { rows = 4: i32, columns = 4: i32 }
      : vector<16xf32> -> vector<16xf32>
   return %0 : vector<16xf32>
 }
 
+// -----
+
+//===----------------------------------------------------------------------===//
+// vector.load
```

Comment:
Ayy, this should be moved after flat_transpose LIT lines.

Reply:
Argh, sorry about that, looked OK on my screen  🤦🏻 Thanks!